travis-ci/run_task_inside_docker.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+IMAGE=$1
+TASK=$2
+COMPONENTS=$3
+
+
+set -xe
+
+# Clean the yum cache
+yum clean all
+
+yum install -y -d0 patch
```

Comment:
Do we need `patch` for tasks other than `rpms`?

Reply:
Yeah, building and testing gsissh.